.github/workflows/ci.yml

```diff
@@ -302,7 +236,7 @@ jobs:
         'project-lint-jobs',
         'project-test-jobs',
       ]
-    if: ${{ always() && github.event_name != 'pull_request' && ! github.event.pull_request.head.repo.fork }}
+    if: ${{ always() && github.event_name != 'pull_request' && ! github.event.pull_request.head.repo.fork  }}
```

Comment:
Doesn't look like properly reverted.

Reply:
I updated this on purpose, to keep the condition of only executing the step on non-forks.